### PR TITLE
Remove rake task

### PR DESCRIPTION
**Pull request by MuriloDalRi**
This changes how the advanced search finder is published. This will now be done using the same rake task as the other finders.

https://trello.com/c/WwLFwsX4/332-remove-publishdocumentfinder-rake-task

### CODE REVIEW

File: doc/publishing-finders.md [hunk 1]
```diff
@@ -6,15 +6,9 @@ not fit the standard specialist document finder pattern. These are:
 - [Advanced search](advanced-search)
```
Comment (leenagupte) [inline doc/publishing-finders.md]: The link to the advanced search docs doesn't work. I think it's missing a `.md`. Can you fix it as well?

**Review by leenagupte — CHANGES_REQUESTED** [review]
Looks good!

Only one minor fix needed to an existing link.

File: doc/publishing-finders.md (hunk 1 above)
Comment (MuriloDalRi) [reply doc/publishing-finders.md]: Fixed

File: doc/publishing-finders.md [hunk 2]
```diff
@@ -6,15 +6,9 @@ not fit the standard specialist document finder pattern. These are:
...
 - Find EU Exit guidance for business (currently in development)
 
-The configuration for these finders can be found in YAML files in
+The configuration for the EU Exit business finder can be found in YAML files in
 the `config/` directory. These config files are a YAML representation
```
Comment (leenagupte) [inline doc/publishing-finders.md]: I think "These config files" should be reworded.

**Review by leenagupte — APPROVED** [review]
LGTM 👍

**Pull request merged**